test/helpers/utils.js

```diff
@@ -0,0 +1,44 @@
+export function getExpectedScorecardLines(pkgName, body) {
+  const { date, score: scorePkg, checks } = body;
+
+  const startOfLines = [
+    "",
+    "                                 OSSF Scorecard",
+    "",
+    mockScorecardCliLine("Repository", pkgName),
+    mockScorecardCliLine("Scan at", date),
+    mockScorecardCliLine("Score", scorePkg),
+    "--------------------------------------------------------------------------------"
+  ];
+
+  const expectedLines = [
```

Comment:
Why declaring startOfLines before? Feel like we have two variables for nothing no?

Reply:
Done